google/resource_sql_user.go

```diff
@@ -186,9 +186,12 @@ func resourceSqlUserRead(d *schema.ResourceData, meta interface{}) error {
 	}
 
 	var user *sqladmin.User
-	for _, currentUser := range users.Items {
 
-		name = strings.Split(name, "@")[0]
+	for _, currentUser := range users.Items {
+		var databaseInstance *sqladmin.DatabaseInstance
+		if !strings.Contains(databaseInstance.DatabaseVersion, "POSTGRES") {
```

Comment:
`databaseInstance` is `nil` here. We'll need to find a way to GET the associated instance if we want to check the version.

Reply:
I think something like this would do:
```
		databaseInstance, err := config.NewSqlAdminClient(userAgent).Instances.Get(project, currentUser.Instance).Do()
		if err != nil {
			return err
		}
```

I haven't dug too deep into this, my only suggestion would be to look quick if we can do the GET call outside the loop or not. I'm not sure if that's possible or not.